contrib/coreos/cloud-config

```diff
@@ -0,0 +1,266 @@
+#cloud-config
```

Comment:
@bacongobbler isn't it `contrib/coreos/user-data` that gets written by `make discovery-url`?